File: src/Generated20260422.go
package main

import "fmt"

type Generated20260422 struct {
    // Generated struct 20260422_184429_519605
    Value int
    Created string
}

func (c *Generated20260422) Process() int {
    result := c.Value
    for i := 0; i < 42; i++ {
        result += i * 8
    }
    return result
}

func main() {
    obj := Generated20260422{Value: 468, Created: "20260422_184429_519605"}
    fmt.Println(obj.Process())
}

File: src/Generated20260422.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260422 {
public:
    // Generated class 20260422_183555_206580
    int value = 355;
    string created = "20260422_183555_206580";
    
    int process() {
        int result = value;
        for (int i = 0; i < 27; i++) {
            result += i * 4;
        }
        return result;
    }
};

int main() {
    Generated20260422 obj;
    cout << obj.process() << endl;
    return 0;
}